bokehjs/src/less/widgets.less

```diff
@@ -45,13 +45,15 @@
 //
 
 /* misc */
-.bk-widget select,
 .bk-widget button {
   min-width: 50%;
 }
 .bk-widget input[type="text"] {
   min-width: 90%;
 }
+.bk-widget select {
+  min-width: 97%;
+}
```

Comment:
Just wondering... Why these (different) numbers? I'd (naively) think that it might prevent things to line up correctly?